terraform/eks/adot-operator/adot_operator.tf

```
@@ -23,6 +23,14 @@ variable "kubeconfig" {
   default = "kubeconfig"
 }
 
+variable "operator_repository" {
+  default = "ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator"
+}
+
+variable "operator_tag" {
+  default = "latest"
```

```suggestion
  default = "latest"
```

Should we remove the default for this variable? This would make it required and thus would error out of this was not provided. 

On the other hand I am not certain that if doing would cause non `adot-operator` tests to error if they do not supply an `operator_tag` variable. We would need to test this to confirm.

---

I think we should keep the default in as it would follow the logic that we had before where the operator repo/tag were hardcoded in `adot-operator-values.yaml`, just adding in more flexibility in it being a variable. Also, like you mentioned, removing it might cause issues elsewhere